README.md

```diff
@@ -91,6 +91,7 @@ Optional features:
 - `backtrace`: include backtrace information in error messages
 - `pyarrow`: conversions between PyArrow and DataFusion types
 - `serde`: enable arrow-schema's `serde` feature
+- `home_dir` : enable support for substituting the tilde character in the file path with the user home directory for the URL table
```

Comment:
I think an optional feature makes sense

Another alternative would be to avoid the dependency entirely. by leaving tilde expansion in datafusion-cli and an API in the core to call it

Reply:
Avoiding the dependency sounds great, but I'm not sure how the API can achieve this. Maybe provide an API to register a URL handling function for it?

Reply:
I remove the feature and leave `substitute_tilde` in the cli. I think avoiding the dependency is a good idea.